data/lua/mapgen_helper.lua

```diff
@@ -24,10 +24,6 @@ local function loc_to_index(map,x,y)
 	return x + 1 + y * map.w
 end
 
-local function index_to_loc(map,index)
```

Comment:
I'm wondering if this function would actually be useful... I see it's not actually used anywhere, but could there be a case where it's needed?

I guess it can always be retrieved from the file history if it's really needed... or it's probably not too difficult to think it up again based on `loc_to_index`...

Reply:
@CelticMinstrel well _currently_ it's dead. Yeah, it'd remain in git history. (Forever.) If there's an exact change you'd like to push before my commit, I can wait of course, but...